packages/components/src/generic_modal/generic_modal.tsx

```diff
@@ -104,7 +106,7 @@ export class GenericModal extends React.PureComponent<Props, State> {
             confirmButton = (
                 <button
                     type='submit'
-                    className={classNames(`GenericModal__button confirm ${this.props.confirmButtonClassName}`, {
```

Comment:
i know this is how it was before. But i think it defeats the purpose of using `classnames` if we are just using strings like this
```js
classNames("GenericModal__button",isConfirmOrDeleteClassName, XXX
```

Reply:
Done 😄